scripts/package

```diff
@@ -60,6 +60,33 @@ initrd \${url}/\${initrd}
 boot
 EOF
 
+GITHUB_REPOSITORY="${GITHUB_REPOSITORY:-rancher-sandbox/os2}"
+UPGRADECHANNEL_DISCOVERY_IMAGE="${UPGRADECHANNEL_DISCOVERY_IMAGE:-quay.io/costoolkit/upgradechannel-discovery:v0.3-4b83dbe}"
```

Comment:
Not sure where to put that honestly. I'd move them up to the Dockerfile, but I noticed in this file there are already other variables set inline, so we should probably go back at this and refactor this script properly separately